.github/workflows/ci.yml

```
@@ -42,8 +40,9 @@ jobs:
       uses: actions/setup-java@v4
       with:
         java-version: '21'
-        distribution: 'temurin'
-    
+        distribution: 'temurin'    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)